app/src/main/java/de/westnordost/streetcomplete/util/KryoSerializer.java

```diff
@@ -58,10 +58,10 @@
 			StringMapEntryAdd.class,
 			StringMapEntryDelete.class,
 			StringMapEntryModify.class,
-			NoteComment.class,
-			NoteComment.Action.class,
+			NoteComment.class, // TODO(Flo): Replace this with Object.class
+			NoteComment.Action.class, // TODO(Flo): Replace this with Object.class
 			Date.class,
-			User.class,
+			User.class, // TODO(Flo): Replace this with Object.class
```

Comment:
Why? It should still be persisted

Reply:
Ah, I understand. Well, as v32.0 will make a clean slate, incompatible serialization is OK in this case as the whole DB is cleared anyway.
(Plus, Kryo will be replaced with kotlinx-serialization anyway)